src/main/java/karlett/task/Task.java

```diff
@@ -0,0 +1,177 @@
+package karlett.task;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+
+public class Task {
+
+    private static int numberOfTasks = 0;
+
+    public static void setFilePath(String filePath) {
+        Task.filePath = filePath;
+    }
+
+    protected static String filePath = "";
+
+    protected String description;
+    protected boolean isDone;
+
+    /* constructor used for user input */
+    public Task(String description) throws IOException {
+        this.description = description;
+        this.isDone = false;
+        increaseNumberOfTasks();
+        appendNewTaskToFile();
+        printNewTaskAddedMessage();
+    }
+
+    /* constructor used for loading file data */
+    public Task(String description, boolean isDone) {
+        this.description = description;
+        this.isDone = isDone;
+        increaseNumberOfTasks();
+    }
+
+    public Task() {
+    }
+
+    protected void printNewTaskAddedMessage() {
+        drawDivider();
+        System.out.println("Karlett now remembers:\n" + "  " + this);
+        if (numberOfTasks == 1) {
+            System.out.println("You have 1 task in the list now meow (((;꒪ꈊ꒪;)))");
+        } else {
+            System.out.println("You have " + numberOfTasks + " tasks in the list now meow (((;꒪ꈊ꒪;)))");
+        }
+        drawDivider();
+    }
+
+    private void appendNewTaskToFile() throws IOException {
+        FileWriter fw = new FileWriter(filePath, true);
+        String textToAppend = "T | 0 | " + description + "\n";
+        fw.write(textToAppend);
+        fw.close();
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "V" : " ");  // mark done task with V
+    }
+
+    public static void increaseNumberOfTasks() {
+        Task.numberOfTasks++;
+    }
+
+    public void markAsDone(int index) throws IOException {
+        this.isDone = true;
+        modifyTaskStatusInFile(index);
+        printMarkAsDoneMessage();
+    }
+
+    private void modifyTaskStatusInFile(int index) throws IOException {
```

Comment:
It would help to briefly describe what this method is doing to modify the task status, to increase code readability.